### ISSUE TITLE
Version 7.0.0 not published to rubygems.org

### ISSUE THREAD

**Issue by KKiti** [joliss/jquery-ui-rails#146]
https://rubygems.org/gems/jquery-ui-rails rubygems.org still shows the latest version of the gem as 6.0.1. Could v7.0.0 be published to rubygems.org, please?

**Comment by skwong2000** [comment]
Currently I can only use it by adding this line in Gemfile:

gem "jquery-ui-rails", :git => 'https://github.com/jquery-ui-rails/jquery-ui-rails.git'

I am also looking forward to 7.0.0 gem in rubygems.org.

**Comment by doconnor-clintel** [comment]
As to why a release isn't published:
![image](https://github.com/jquery-ui-rails/jquery-ui-rails/assets/86343144/fcc95ab5-b896-43d8-acc8-f03deace44f7)

**Comment by massongit** [comment]
📝The link to above comment: https://github.com/jquery-ui-rails/jquery-ui-rails/pull/139#issuecomment-1438948513

**Comment by massongit** [comment]
@Borzik @rosenfeld @joliss jquery-ui-rails v7.0.0 contains security updates for CVE-2022-31160, CVE-2021-41182 and CVE-2021-41183.
Therefore, please publish to rubygems.org.

**Comment by dacook** [comment]
Another way to set in your gemfile, which will lock it to a particular version:

```
gem 'jquery-ui-rails', github: 'jquery-ui-rails/jquery-ui-rails', tag: 'v7.0.0'
```

**Comment by jeffgran-dox** [comment]
We [forked](https://github.com/doximity/jquery-ui-rails/) and published a new version of this gem with jquery-ui 1.13.2 here, in case this might help someone: https://rubygems.org/gems/jquery-ui-rails-dox-fork

**Comment by fmborghino** [comment]
More folks will hit this (anyone running `bundle audit`) as the ruby-advisory-db project added issues that require jquery-ui-rails >=7.0.0 yesterday: https://github.com/rubysec/ruby-advisory-db/issues/747 - we're going with pointing at the github tag for now - fingers crossed that this gets pushed to rubygems.org. Thanks for the project!

**Comment by dennisvandehoef** [comment]
Releasing 7.0.0 only helps if the gem will be maintained after releasing it.

Looking at an even older issue https://github.com/jquery-ui-rails/jquery-ui-rails/issues/140, I think we should not ask the maintainers to keep maintaining this if they moved on.

Maybe we need new maintainer, though simply releasing version 7.0.0 will not help us here. Latest when you want the update to jquery-ui 1.13.2 (as @jeffgran-dox mentioned) we have the issue again.

**Comment by tcostermans** [comment]
Any updates on this issue?
Trying to get rid of the bunlde audit vulnerability without pointing to the fork or a github tag.

**Comment by prpetten** [comment]
@jeffgran-dox Are you going to be maintaining your fork for the foreseeable future?

**Comment by jeffgran-dox** [comment]
@prpetten we have money-making production systems still using jquery and jquery-ui in some places and no appetite to migrate them, so yes we have vested interest in maintaining this as long as that is the case. I cannot make any kind of specific guarantees of course but that's why we decided to fork and publish this.

**Comment by prpetten** [comment]
@jeffgran-dox I actually tracked down one of the previous gem owners of this gem via email, and he would be willing to give ownership to someone who would maintain it.  Would you be willing to take over the maintenance and publishing of the original gem here, and merge your changes back in here?

**Comment by prpetten** [comment]
Working on getting @jeffgran-dox access to take over maintenance and publishing of the gem moving forward.

**Comment by prpetten** [comment]
7.0.0 gem is now published.

**Comment by joshuapinter** [comment]
@prpetten @jeffgran-dox Legends! Thank you for doing this. This is what "community" is all about. 🙏

**Comment by c960657** [comment]
> @Borzik @rosenfeld @joliss jquery-ui-rails v7.0.0 contains security updates for CVE-2022-31160, CVE-2021-41182 and CVE-2021-41183. Therefore, please publish to rubygems.org.

AFAICT CVE-2022-31160 was **not** fixed in v7.0.0. This CVE affects jquery-ui < 1.13.2, but jquery-ui-rails 7.0.0 only includes jquery-ui 1.13.0.
